app/assets/javascripts/discourse/templates/group-index.hbs

```diff
@@ -64,7 +64,8 @@
                     removeMember=(action "removeMember")
                     makeOwner=(action "makeOwner")
                     removeOwner=(action "removeOwner")
-                    member=m}}
+                    member=m
+                    group=model}}
```

Comment:
Not sure if there's a better way to do this. group=model is only useful from a plugin's perspective 🤔

Reply:
maybe as a metadata hash ?

Reply:
Maybe I should rename it to `pluginMetadata`?

Reply:
I must be missing something, what's wrong with it being called `group`? That seems like a simple and descriptive variable name to me

Reply:
My intention here is to emphasize this is needed for plugins and avoid it being deleted because someone thought is not being used.

Reply:
Ah, I see. Maybe just a comment next to the line? `{{!-- group parameter is used by plugins --}}`. If it's renamed then I like the idea of `pluginMetadata` - that's super clear :+1: